### ISSUE TITLE
Optional async eager loading

### ISSUE THREAD

**Issue by elpete** [coldbox-modules/quick#55]
Each `with` relationship should be able to be parallelized. Add an optional argument or setting to do this.

**Comment by elpete** [comment]
Waiting on ColdBox 6

**Comment by elpete** [comment]
Implemented by [PR #339](https://github.com/coldbox-modules/quick/pull/339), now merged into `next`. Opt in with `.with( [ "author", "tags" ], true )`. It uses a bounded ColdBox executor, isolated worker contexts, and caller-side parent matching; worker failures/timeouts propagate. Adobe ColdFusion, active transactions, and a single top-level branch retain sequential execution.

Local full suites passed 674 tests in each Lucee 6 null mode and 676 on BoxLang 1, with only existing skips. [Configuration and behavior](https://github.com/coldbox-modules/quick/blob/e9399de470afb802ecc049ca97139d22ce0e62af/README.md#optional-parallel-eager-loading). This is available on next and is not yet a new Quick package release.
